Modules/Microsoft365DSC/Examples/Resources/AADGroup/4-License.ps1

```diff
@@ -0,0 +1,43 @@
+<#
```

Comment:
We should only ever have 3 examples max: Create, Update, Remove. This drives our integration tests platform and adding a 4th one will cause issue.